src/backends/tensorflow/base_backend.cc

```diff
@@ -409,6 +409,11 @@ ReadStringOutputTensor(
     std::vector<Scheduler::Payload>* payloads)
 {
   size_t tensor_element_idx = 0;
+  bool cuda_copy = false;
+
+#ifdef TRTIS_ENABLE_GPU
+  cudaStream_t stream = nullptr;
```

Comment:
There should be a `stream_` member in context. Just move this function to the context class and use it. Or if you don't want to move function around, you still need to create the stream, otherwise the default stream will be used